src/userbehavior/.gitignore

```diff
@@ -0,0 +1,163 @@
+# emacs recovery files
```

Comment:
IMHO the userbehavior should not have an extra .gitignore file. Can we remove this?

Reply:
I do not know if it still needed at all and if it can just be removed as a whole. It was copied with the userbehavior code. I guess the contents of this .gitignore file can be moved to the main .gitignore file?